hilti/toolchain/src/compiler/cfg.cc

```diff
@@ -69,6 +70,32 @@
 
 namespace hilti::detail::cfg {
 
+std::deque<GraphNode> CFG::postorder() const {
+    std::deque<GraphNode> sorted;
+
+    std::unordered_set<NodeId> visited;
+
+    std::function<void(NodeId)> dfs_visit = [&](NodeId node_id) {
+        if ( visited.contains(node_id) )
+            // We either already added this or will add this.
```

Comment:
Outdated. Since the code is simpler now the comment doesn't add much, so I'd probably drop it altogether. If we keep it lets update it.

```suggestion
            // We already added this.
```